env.example

```diff
@@ -35,6 +35,19 @@ LOG_LEVEL=debug
 # LOG_FILE is used to set a destination path to write logs. Works in production mode only.
 LOG_FILE=
 
+# Used to set if Pino should log are indexed to ElasticSearch, LOG_ELASTIC=1 to enable, if LOG_FILE and LOG_ELASTIC is set,
+# LOG_ELASTIC will take priority
+LOG_ELASTIC=
+# Following are used to set up Pino-Elastic
+# Index name for logs to be stored under
+INDEX_NAME=
+# Determines consistency of the write, valid values are: one, quorum or all
+CONSISTENCY=
+# Elasticsearch Version
+ELASTIC_VERSION=
+# The number of bytes for each bulk insert
+FLUSH_BYTES=
+
```

Comment:
I think we should have default values here like we have in all the other variables.

Reply:
I left default values in the other `logger.js` file. I have no idea which is the preferred one.

Reply:
I think both should have default values, for consistency

Reply:
Sure, I have no issues with that